.github/workflows/data-product-example-2-poc.yaml

```
@@ -0,0 +1,54 @@
+---
+name: POC - zip and upload _example_2
```

```suggestion
name: Data Product - example_2 - PoC
```
as per https://github.com/ministryofjustice/data-platform-products/blob/main/.github/workflows/data-product-example-2-test.yaml#L2